internal/commands/root_test.go

```diff
@@ -72,6 +74,75 @@ func createASTTestCommand() *cobra.Command {
 		projectsMockWrapper,
 		resultsMockWrapper,
 		risksOverviewMockWrapper,
+		scsScanOverviewMockWrapper,
+		authWrapper,
+		logsWrapper,
+		groupsMockWrapper,
+		gitHubWrapper,
+		azureWrapper,
+		bitBucketWrapper,
+		nil,
+		gitLabWrapper,
+		bflMockWrapper,
+		prMockWrapper,
+		learnMoreMockWrapper,
+		tenantConfigurationMockWrapper,
+		jwtWrapper,
+		scaRealtimeMockWrapper,
+		chatWrapper,
+		featureFlagsMockWrapper,
+		policyWrapper,
+		sastMetadataWrapper,
+		accessManagementWrapper,
+		byorWrapper,
+	)
+}
+
+func createASTTestCommandWithScs(scsScanned, scsScanPartial, scorecardScanned bool) *cobra.Command {
```

Comment:
Since we have our summary in the console report we created tests that test several possible interactions. You can see we have several parameters as inputs in that method. They are used in the mocks to include our report/ parts of our report. We then test this in the output of the console in unit tests. If we included this in the existing createASTTestCommand method, we would need to change all existing 24 usages of that method.

Reply:
you can do it as it was done in FeatureFlagsMockWrapper
but if it is not enough for your tests, you can at least overload the function...

Reply:
Followed the example of FeatureFlagsMockWrapper